### ISSUE TITLE
Documentation using session storage Redis

### ISSUE THREAD

**Issue by john-everden** [FoalTS/foal#1144]
I'm wondering if the documentation should be updated? There is no mention under https://foalts.org/docs/authentication-and-access-control/session-tokens of having to call commit to sessions but it doesn't appears sessions are stored in Redis unless commit is called?
```typescript
    ctx.session = await createSession(this.store);
    ctx.session.setUser({id: 'test'});
    await ctx.session?.commit();
```

**Comment by LoicPoullain** [comment]
> sessions are stored in Redis unless commit is called

This is true, sessions are comitted using the the `commit()`. But normally this should be done automatically with `@UseSessions` when the `ctx.session` property is defined.

Is your controller or controller method decorated with the hook `@UseSessions`?

**Comment by john-everden** [comment]
Sorry that was it :). Thanks for the help!
